From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Oleg <balbes-150@yandex.ru>
Date: Mon, 27 Dec 2021 08:27:01 +0100
Subject: [ARCHEOLOGY] fix kernel legacy for firefly-rk3399 (#3369)

> X-Git-Archeology: - Revision e453e4dfbad6f9ceb5895b3f7812ae28c5d38db5: https://github.com/armbian/build/commit/e453e4dfbad6f9ceb5895b3f7812ae28c5d38db5
> X-Git-Archeology:   Date: Mon, 27 Dec 2021 08:27:01 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: fix kernel legacy for firefly-rk3399 (#3369)
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/Makefile           | 1 +
 arch/arm64/boot/dts/rockchip/rk3399-firefly.dts | 7 +++++++
 2 files changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/Makefile b/arch/arm64/boot/dts/rockchip/Makefile
index 9dcebdaff2ff..3edaefe5593b 100644
--- a/arch/arm64/boot/dts/rockchip/Makefile
+++ b/arch/arm64/boot/dts/rockchip/Makefile
@@ -87,6 +87,7 @@ dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev3-android-mipi-edp.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev3-cros.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-firefly-android.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-firefly-linux.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-firefly.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-roc-pc-mezzanine.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-roc-pc-plus.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-roc-pc.dtb
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-firefly.dts b/arch/arm64/boot/dts/rockchip/rk3399-firefly.dts
new file mode 100644
index 000000000000..f9f0526ce22b
--- /dev/null
+++ b/arch/arm64/boot/dts/rockchip/rk3399-firefly.dts
@@ -0,0 +1,7 @@
+/dts-v1/;
+#include "rk3399-firefly-linux.dts"
+
+/ {
+	compatible = "rockchip,rk3399-firefly", "rockchip,rk3399";
+
+};
-- 
Armbian

